bundles/org.openhab.binding.mybmw/src/main/java/org/openhab/binding/mybmw/internal/handler/VehicleHandler.java

```diff
@@ -951,6 +993,25 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                     logger.debug("Cannot handle command {}, channel {} in group {} not a command channel",
                             command.toFullString(), channelUID.getAsString(), group);
             }
+        } else if (command instanceof OnOffType) {
+            if (CHANNEL_GROUP_UPDATE.equals(group) && OnOffType.ON.equals(command)) {
+                // triggering the update of the respective channel
+                switch (channelUID.getIdWithoutGroup()) {
+                    case STATE_UPDATE:
+                        updateVehicleStatus();
+                        break;
+                    case CHARGING_UPDATE:
+                        updateCharging();
+                        break;
+                    case IMAGE_UPDATE:
+                        updateImage();
+                        break;
+                    default:
+                        break;
+                }
+                // immediately set the switch back to off
+                updateChannel(group, channelUID.getIdWithoutGroup(), OnOffType.OFF, null);
```

Comment:
This is not needed with `<autoUpdatePolicy>veto</autoUpdatePolicy>` in the channel type definition.

Reply:
Ok I didn't know that, will test it

Reply:
I changed the XML config together with the Java code in one commit